packages/lesswrong/components/linkPreview/PostLinkPreview.tsx

```diff
@@ -794,6 +794,57 @@ const ArbitalPreviewComponent = registerComponent('ArbitalPreview', ArbitalPrevi
   styles: arbitalStyles
 })
 
+const estimakerStyles = (theme: ThemeType): JssStyles => ({
+  iframeStyling: {
+    width: 560,
+    height: 405,
+    border: "none",
+    maxWidth: "100vw",
+  },
+  link: linkStyle(theme),
+});
+
+const EstimakerPreview = ({classes, href, innerHTML, id}: {
+  classes: ClassesType;
+  href: string;
+  innerHTML: string;
+  id?: string;
+}) => {
+  const { AnalyticsTracker, LWPopper } = Components;
+  const { anchorEl, hover, eventHandlers } = useHover();
+
+  // test if fits https://metaforecast.org/questions/embed/[...]
+  const isEmbed = /^https?:\/\/estimaker\.app\/_\/.+$/.test(href);
+
+  // test if it fits https://manifold.markets/questions/[...] instead
+  const [, questionId] = href.match(/^https?:\/\/metaforecast\.org\/questions\/([\w-]+)/) || [];
+
+  if (!isEmbed && !questionId) {
+    return (
+      <a href={href}>
+        <span dangerouslySetInnerHTML={{ __html: innerHTML }} />
+      </a>
+    );
+  }
+
+  const url = isEmbed ? href : `https://metaforecast.org/questions/embed/${questionId}`;
```

Comment:
This needs to be changed from metaforecast right?

Reply:
This logic wasn't needed in our case. Thanks for flagging.